packages/playwright/package.json

```diff
@@ -53,6 +52,8 @@
     "express": "^4.18.1",
     "mocha": "^10.0.0",
     "nyc": "^15.1.0",
+    "playwright": "^1.0.0",
+    "playwright-core": "^1.25.2",
```

Comment:
`playwright-core` probably needs to be a dependency as we are utilising functions from Playwright rather than just exposing their types